draft-ietf-tls-tls13.md

```diff
@@ -3753,32 +3756,35 @@ Note: In the case of the CCM mode of AES, two variations exist: "CCM_8" which
 uses an 8-bit authentication tag and "CCM" which uses a 16-bit authentication
 tag. Both use the default hash, SHA-256.
 
-In addition to authenticated cipher suites, completely anonymous Diffie-Hellman
-cipher suites exist to provide communications in which neither party is
-authenticated. This mode is vulnerable to man-in-the-middle attacks and is
-therefore unsafe for general use. These cipher suites MUST NOT be used by TLS
-implementations unless the application layer has specifically requested to allow
-anonymous key exchange. Anonymous key exchange may sometimes be acceptable, for
-example, to support opportunistic encryption when no set-up for authentication is
-in place, or when TLS is used as part of more complex security protocols that have
-other means to ensure authentication. The following specifications provide "DH_anon"
-key exchange cipher suites:
-AES-GCM [RFC5288], ARIA-GCM [RFC6209], and CAMELLIA-GCM [RFC6367].
-
 All cipher suites in this section are specified for use with both TLS 1.2
 and TLS 1.3, as well as the corresponding versions of DTLS.
 (see {{backward-compatibility}})
 
-Note that using non-anonymous key exchange without actually verifying the key
-exchange is essentially equivalent to anonymous key exchange, and the same
-precautions apply. While non-anonymous key exchange will generally involve a
-higher computational and communicational cost than anonymous key exchange, it
-may be in the interest of interoperability not to disable non-anonymous key
-exchange when the application layer is allowing anonymous key exchange.
-
 New cipher suite values are assigned by IANA as described in
 {{iana-considerations}}.
 
+### Unauthenticated Operation
+
+Previous versions of TLS offered explicitly unauthenticated cipher suites
+base on anonymous Diffie-Hellman. These cipher suites have been deprecated
+in TLS 1.3. However, it is still possible to negotiate cipher suites
+that do not provide verifiable server authentication by serveral methods,
+including:
+
+- Raw public keys {{RFC7250}}.
+- Using a public key contained in a certificate but without
+  validation of the certificate chain or any of its contents.
+
+Either technique used alone is are vulnerable to man-in-the-middle attacks
+and therefore unsafe for general use. However, it is also possible to
+bind such connections to an external authentication mechanism via
+out-of-band validation of the server's public key, trust on first
+use, or channel bindings {{RFC5929}} [[NOTE: TLS 1.3 needs a new
+channel binding definition that has not yet been defined.]]
+If no such mechanism is used, then the connection has no protection
+against active mon-in-the-middle attack; applications MUST NOT use TLS
```

Comment:
1) These are really just two sentences. You can just drop the semicolon and replace it with a period and subsequent capitalization.
2) "mon-in-the-middle" -> "man-in-the-middle"